horace_core/sqw/@DnDBase/equal_to_tol_internal.m

```diff
@@ -0,0 +1,31 @@
+function [ok, mess] = equal_to_tol_internal(w1, w2, name_a, name_b, varargin)
```

Comment:
If this is internal, should it be in a `private` folder, or does that mess up the "protected"-ness?

Reply:
It's also common (in Horace) to suffix private functions with an underscore, although I realise this isn't done consistently

Reply:
The function must be `protected` in order that `equal_to_tol` in the base class can access it.